src/Containers/RectangleTable/RectangleTable.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { occupancyStatusTypes } from '../CircleTable2/CircleTable2';
```

Comment:
I'm not entirely sure why this file is here, since it is not mine. There isn't a CircleTable2 module, so I'd presume it's CircleTable but I'm not sure.